tests/acceptance/image-set/lion-search-test.js

```diff
@@ -0,0 +1,75 @@
+import Ember from 'ember';
+import startApp from '../../helpers/start-app';
+import {stubRequest} from '../../helpers/fake-server.js';
+import { stubGetImageSet,
+         stubGetOrganizations,
+         stubImageSetJSON,
+         stubGetUser,
+         stubGetLions } from '../../helpers/fake-requests';
+
+var application;
+
+module('Acceptance: ImageSetLionSearch', {
+  setup: function() {
+    application = startApp();
+    stubGetOrganizations();
+    stubGetLions();
+    stubGetImageSet(25);
+    stubGetUser();
+  },
+  teardown: function() {
+    Ember.run(application, 'destroy');
+  }
+});
+
+test('visiting /image-set/lion-search', function() {
+  expect(6);
+
+  // visit image (set) stub request
+  signInAndVisit('/image-set/25');
+
+  andThen(function(){
+    click('.lion-search-button');
```

Comment:
this doesn't have to be in an `andThen`. ember's async test helpers will correctly join up the `signInAndVisit` with the `click`.